src/lib/seam/use-seam-styles.ts

```diff
@@ -1,21 +1,31 @@
 import { useEffect } from 'react'
 
-// TODO figure out the version automatically
-const cssUrl = 'https://www.unpkg.com/@seamapi/react@1.1.0/index.min.css'
+import version from 'lib/version.js'
+
+const origin = 'https://react.seam.co'
```

Comment:
If we always use the css file in the release / bundle, wouldn't it always have the correct version?

Reply:
No because we are injecting a URL that contains the version number which must match the version of the package that is being used by the user. I'm not sure if that explanation is more clear, but it's correct.

Reply:
> No because we are injecting a URL that contains the version number 

Yep, was wondering what the benefits of injecting via URL vs. just including what is bundled with the package are?

Reply:
Ah, there is a pretty long internal discussion about this. The short version is that having it inject the fonts and the styles simplifies adoption and removes all the extra steps you need to use this. 

It's still possible to opt out of the HTML injection and do it the other way for users who prefer that.

Reply:
Was actually thinking of dynamically importing the css via relative paths, but of course that wouldn't work because this is runtime.

What if we always include the CSS, but just change the class name? Won't have to deal with version numbers, and no dynamic loading necessary?

```tsx
const class = disableStyles ? '' : 'seam-components'

return (<div className={class}>...</div>)
```

Reply:
> What if we always include the CSS

I'm not sure I understand this. With this update we are always including the css unless the user opts out. I'd prefer not to inject the via an inline style tag since that bloats the js bundle size and cannot take advantage of caching the same way (among other things).

Reply:
>  I'd prefer not to inject the via an inline style tag

Yep, I mean we'd be able to avoid that. Here's a PR:

https://github.com/seamapi/react/pull/191